scripts/convert_data_format.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+"""Convert a recorded pickle file to the new format."""
+import argparse
+import json
+import logging
+import pathlib
+import pickle
+import sys
+
+
+def convert_record_1to2(record):
+
+    del record["num_subjects"]
+    del record["subjectNames"]
+
+    subjects = {}
+    for key in list(record.keys()):
+        if key.startswith("subject_"):
+            subject = record[key]
+            name = subject["name"]
+
+            # delete obsolete stuff
+            del subject["name"]
+            del subject["global_rotation"]["eulerxyz"]
+            del subject["global_rotation"]["helical"]
+            del subject["global_rotation"]["matrix"]
+
+            subjects[name] = subject
+
+            # remove the subject_# entry
+            del record[key]
+
+    record["subjects"] = subjects
+    record["format_version"] = 2
+
+    return record
```

Comment:
if record is mutable, do we need the return ?

Reply:
I think it's nicer this way as with `record = convert(record)` it is more obvious that `record` is modified